src/components/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -0,0 +1,585 @@
+    id: "art_history",
+    name: "Art History",
+  },
+  {
+    id: "design",
+    name: "Design",
+  },
+  {
+    id: "business",
+    name: "Business",
+  },
+  {
+    id: "advertising",
+    name: "Advertising",
+  },
+  {
+    id: "government_and_law",
+    name: "Government and Law",
+  },
+  {
+    id: "civil_rights",
+    name: "Civil Rights",
+  },
+  {
+    id: "history",
+    name: "History",
+  },
+  {
+    id: "atlases",
+    name: "Atlases",
+  },
+  {
+    id: "cartography",
+    name: "Cartography",
+  },
+  {
+    id: "geology",
+    name: "Geology",
+  },
+];
+
+const withChildrenItems = [
+  {
+    id: "art",
+    name: "Art",
+  },
+  {
+    id: "architecture",
+    name: "Architecture",
+  },
+  {
+    id: "design",
+    name: "Design",
+    children: [
+      {
+        id: "fashion",
+        name: "Fashion",
+      },
+      {
+        id: "ux",
+        name: "User Experience",
+      },
+      {
+        id: "tecture",
+        name: "Architecture",
+      },
+      {
+        id: "home",
+        name: "Home",
+      },
+    ],
+  },
+  {
+    id: "business",
+    name: "Business",
+  },
+  {
+    id: "education",
+    name: "Education",
+  },
+  {
+    id: "games",
+    name: "Games",
+  },
+];
+
+const withDisabledItems = [
+  {
+    id: "art",
+    name: "Art",
+  },
+  {
+    id: "architecture",
+    name: "Architecture",
+    isDisabled: true,
+  },
+  {
+    id: "art_history",
+    name: "Art History",
+  },
+  {
+    id: "design",
+    name: "Design",
+    children: [
+      {
+        id: "fashion",
+        name: "Fashion",
+        isDisabled: true,
+      },
+      {
+        id: "ux",
+        name: "User Experience",
+        isDisabled: true,
+      },
+      {
+        id: "tecture",
+        name: "Architecture",
+        isDisabled: false,
+      },
+      {
+        id: "home",
+        name: "Home",
+        isDisabled: false,
+      },
+    ],
+  },
+  {
+    id: "business",
+    name: "Business",
+  },
+];
+
+const withDisabledAllChildrenItems = [
+  {
+    id: "art",
+    name: "Art",
+  },
+  {
+    id: "architecture",
+    name: "Architecture",
+    isDisabled: true,
+  },
+  {
+    id: "art_history",
+    name: "Art History",
+  },
+  {
+    id: "design",
+    name: "Design",
+    children: [
+      {
+        id: "fashion",
+        name: "Fashion",
+        isDisabled: true,
+      },
+      {
+        id: "ux",
+        name: "User Experience",
+        isDisabled: true,
+      },
+      {
+        id: "tecture",
+        name: "Architecture",
+        isDisabled: true,
+      },
+      {
+        id: "home",
+        name: "Home",
+        isDisabled: true,
+      },
+    ],
+  },
+  {
+    id: "business",
+    name: "Business",
+  },
+];
+
+const meta: Meta<typeof MultiSelect> = {
+  title: "Components/Form Elements/MultiSelect",
+  component: MultiSelect,
+  decorators: [withDesign],
+  argTypes: {
+    buttonText: {
+      description: "The button text rendered within the MultiSelect.",
```

Comment:
Once the MultiSelectProp is changed to `interface` all descriptions can be removed.